pkg/daemon/ceph/config/config_test.go

```diff
@@ -54,13 +54,19 @@ func TestCreateDefaultCephConfig(t *testing.T) {
 		},
 	}
 
-	cephConfig := CreateDefaultCephConfig(context, clusterInfo, "/var/lib/rook1")
+	cephConfig, err := CreateDefaultCephConfig(context, clusterInfo, "/var/lib/rook1")
+	if err != nil {
```

Comment:
instead of checking for nil, a common pattern is:
```
assert.Nil(t, err)
```
The err will show in the details of the assert message by default, or you can add more details as a third param

Reply:
The current code uses this pattern everywhere so I just re-used it, do you want me to change all of them then?

Reply:
It's fine if that's the current pattern, we can save it for another day